wordpress-mobile/WordPress-iOS
# Notifications: Fixing blavatar image

### Details:
This PR fixes an issue that caused Blavatar images not to be properly downloaded / displayed.

Fixes #8190

Needs Review: @koke 

Sir, may i bug you with a quick patch?. I've got ~50 WPcom users, please, let me know if i can help you triggering a New Follower notification!


### To test:
1. Open WordPress.com and log into a sample site
2. Open the site's settings (thru the calypso admin) and upload a new Blavatar
3. Receive a New Follower notification
4. Open the Follower notification in iOS and expand it's details

Verify that the Top Header icon (in the Notification Details) properly displays a blavatar image.
It would appear that, now, Blavatars may be stored not just in gravatar.com.
